portfolio/src/main/java/com/google/sps/servlets/ReadGameServlet.java

```diff
@@ -43,24 +51,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         List<Game> games = new ArrayList<>();
         for(Entity entity :results.asIterable()){
-            game = new Game();
-            game.white = (String)entity.getProperty("white");
-            game.black = (String)entity.getProperty("black");
-            game.result = (String)entity.getProperty("result");
-            parseMoves((String)entity.getProperty("moves"));
-            
-            games.add(game);
+            String w = (String)entity.getProperty("white");
+            String b = (String)entity.getProperty("black");
+            if(w.equals(searchTerm) || b.equals(searchTerm)){
```

Comment:
Just to make sure, this is supposed to be a way to search for all games for a player specified by searchTerm, right?